build-scripts/prepare_environment.sh

```diff
@@ -75,17 +75,11 @@ FULL_BUILD_VERSION="${HADOOP_VERSION}-${RELEASE}"
 
 MAVEN_ARGS="$MAVEN_ARGS -Dhadoop.version=$MAVEN_VERSION"
 
-#
-# Dump generated env vars into rc file
-#
-
-cat >> "$BUILD_COMMAND_RC_FILE" <<EOF
-export MAVEN_ARGS='$MAVEN_ARGS'
-export SET_VERSION='$MAVEN_VERSION'
-export HADOOP_VERSION='HADOOP_VERSION'
```

Comment:
This seems like a typo to me that should have originally been "$HADOOP_VERSION", I've kept it as just a static string to mimic the old behaviour but I'll change it once I'm validated that it should be parsed as an env variable not a static string..

Reply:
I think this was copied from the hbase build, and i dont think it's necessary here. This seems especially true since the raw string 'HADOOP_VERSION' is not a useful value. Let's just remove this and the echo below

Reply:
Done 👍